provisioning/miniprov/hare_mp/cdf.py

```diff
@@ -0,0 +1,92 @@
+import os.path as P
+import subprocess as S
```

Comment:
why not just do import `subprocess` and use as it is?

Reply:
Just a matter of readability. instead of `S.PIPE` and `S.Popen` one would have to write and read ubiquitous `subprocess.PIPE` and `subprocess.Popen`. Although the word "subprocess" doesn't bring that much of value to the reader anyway.